docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/parameters.yaml

```diff
@@ -0,0 +1,11 @@
+kernel:
```

Comment:
I think this was by mistake. As I can see EDGE_DETECTION doesn't accept any parameters